README.md

```diff
@@ -42,16 +79,29 @@ prompt(
 );
 ```
 
-## Props
+### Options
+The third argument is an object. It can have any of these keys:
+
+| Key                 | Description                                                                       | Type    | Default                                              |
+|---------------------|-----------------------------------------------------------------------------------|---------|------------------------------------------------------|
+| type                | Text input type: `'numeric', 'secure-text', 'phone-pad', 'email-address'`         | String  | 'default'                                            |
+| cancelable          |                                                                                   | Boolean | true                                                 |
+| defaultValue        | Default input value                                                               | String  |                                                      |
+| placeholder         |                                                                                   | String  |                                                      |
+| style               | `'default', 'shimo', 'cust'`                                                      | String  | 'default'                                            |
+| disableFullScreenUI | When in landscape mode, don't use fullscreen                                      | Boolean | false                                                |
```

Comment:
Typo, should be disableFull**s**creenUI

Reply:
Thanks very much for going through this and catching it. Fixing right now.